keyboards/xd60/keymaps/styr/keymap.c

```diff
@@ -0,0 +1,62 @@
+#include QMK_KEYBOARD_H
+#include "action_layer.h"
+
+#define _BL 0
+#define _FL 1
+#define _LL 2
+// #define _RS 3
+
+#define _______ KC_TRNS
```

Comment:
These are already included when you do `#include QMK_KEYBOARD_H`

```suggestion

```

Reply:
These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31